tests/test_cli.py

```diff
@@ -0,0 +1,85 @@
+import click
+from click.testing import CliRunner
+from getgist.__main__ import run_getgist, run_getmy
+from getgist.__main__ import run_putgist, run_putmy, run_lsgists, run_mygists
+from getgist.__main__ import GETGIST_DESC, GETMY_DESC, PUTGIST_DESC, PUTMY_DESC, LSGISTS_DESC, MYGISTS_DESC
+
+import pytest
+
```

Comment:
This import is also unused, isn't it?

```suggestion
```

Reply:
This import is used for running tests with `mocker` and `monkeypatch`, so its necessary!

Reply:
Can you remove this import, run the tests and share the full error?

`mocker` and `monkeypatch` should  **not** require `import pytest` since they are injected as function arguments.

Reply:
Fixed!